apps/api/src/app/agents/usecases/consume-telegram-mobile-link/consume-telegram-mobile-link.usecase.ts

```
@@ -0,0 +1,118 @@
+import {
+  BadRequestException,
+  ConflictException,
+  Injectable,
+  Logger,
```

Biome (2.4.15)

[error] Please use the PinoLogger from `@novu/application-generic` instead

(lint/style/noRestrictedImports)